src/app/components/profile/profile.component.scss

```diff
@@ -78,19 +73,138 @@
 		}
 	}
 }
-
 @include screen-medium {
 	.profile-container {
 		.card {
 			width:70%;
 		}
 	}
 }
-
 @include screen-small {
 	.profile-container {
 		.card {
 			width:90%;
 		}
 	}
 }
+.profile-sidebar {
+    position: fixed;
+    z-index: 9;
+    font-size: 14px;
+    /*padding:0px  30px;*/
+    min-width: 200px;
+    min-height: 100vh;
+    background-color: $dust-gray;
+    top: 0;
+    left: -200px;
+    text-align: center;
+    color: #fff;
+    /*transition:all 0.2s ease-in-out;*/
+    ul {
+        margin-top: 20px;
+        text-align: left;
+        li {
+            padding: 10px 0px;
+        }
+    }
+    input[type="checkbox"] + label {
+        font-size: 12px;
+    }
+}
+.token-width {
+    width: 400px;
+}
+.profile-edit-group {
+	margin-bottom: 50px;
+	margin-right: -20px;
+}
+
+.pass-title {
+    margin-bottom: 30px;
+}
+
+.back-profile-group {
+    margin-top: 60px;
+}
+
+img.profile-pic {
+    height: 80px;
+}
+
+.user-info {
+    li {
+     margin-bottom: 36px;
+  }
+   margin-top: 20px;
+ }
+ .user-urls {
+      li {
+		margin-bottom: 33px;
+	  }
+	  margin-bottom: 20px;
+	  margin-top: 25px;
+ }
+.container-fluid{
```

Comment:
Please give a space ` ` after class name here and at other places.

Reply:
@Suryansh5545 Also I mean to give a space before `{` just after class name in this case after `container-fluid`. It should look like this `.container-fluid {`

Reply:
done sir